[#69455] 17.0 Meetings module changes  https://community.openproject.… (#21353)
opf/openproject: docs/user-guide/meetings/recurring-meetings/README.md

Within the same menu you also have the following options:

- Download meeting series as iCalendar event

- Send email to all participants

> - End meeting series (this option is only displayed of no occurrence of the meeting series has been closed before)

Reviewer: Hmm there's a typo here, but the whole thing is incorrect. It's visible only when the series itself hasn't been ended before, not anything to do with occurrences. But since that's a bit of an edge case (because ideally, once ended, users aren't gonna extend and 'restart' the same series), I think you can just skip the explanatory text. 

What do you think?

Author: @mrmir I was a bit fuzzy on this one. It does not appear in the screenshot that I took, so I wanted to explain that it might and under which conditions. 
how about this
- End meeting series (this option is only displayed if the series has not been ended before)

Reviewer: Sure, that works

- Delete meeting series

  

### Edit a recurring meeting occurrence

To edit a single meeting within recurring meeting series you have to open it first by clicking the **Open** button next to the meeting. It will then be displayed under *Agenda opened* section on the recurring meeting index page, where you can click the meeting date and time. 

> ![Select a meeting occurrence on a recurring meetings series index page](opEnproject_userguide_meetings_edit_meeting_occurence_link.png)

Author: ```suggestion
![Select a meeting occurrence on a recurring meetings series index page](openproject_userguide_meetings_edit_meeting_occurence_link.png)
```

Within the same menu you also have the following options:

- Download meeting series as iCalendar event

- Send email to all participants

> - End meeting series (this option is only displayed of no occurrence of the meeting series has been closed before)

Author: ```suggestion
- End meeting series (this option is only displayed if the series has not been ended before)
```
